kotlin-typescript/src/jsMain/generated/typescript/Classifications.kt

```diff
@@ -2,9 +2,7 @@
 
 package typescript
 
-import js.array.ReadonlyArray
```

Comment:
Could we restore imports for `ReadonlyArray`?

Reply:
Import hosting is not safe without AST analysis and  Karakum is not good tool to provide AST modifications for Kotlin code.